src/main/java/redis/clients/jedis/ClusterCommandObjects.java

```diff
@@ -15,7 +15,7 @@
 public class ClusterCommandObjects extends CommandObjects {
 
   @Override
-  protected ClusterCommandArguments commandArguments(ProtocolCommand command) {
+  public ClusterCommandArguments commandArguments(ProtocolCommand command) { // TODO: restrict access??
```

Comment:
TODO?

Reply:
Yes. For this PR, I made this less restricted (public). I hope to check if it is possible to make more restricted (i.e. protected) again.